lib/ansible/plugins/connection/winrm.py

```diff
@@ -276,7 +285,7 @@ def _kerb_auth(self, principal, password):
             password = ""
 
         self._kerb_ccache = tempfile.NamedTemporaryFile()
-        display.vvvvv("creating Kerberos CC at %s" % self._kerb_ccache.name)
+        display.debug("creating Kerberos CC at %s" % self._kerb_ccache.name)
```

Comment:
Keep as before

Reply:
Is this really something we want to expose to the user, ever ? (i.e. when not debugging ?)
I am not convinced though.

Reply:
But I am not using Kerberos, so I'll leave it up to my successor ;-)
Fixed.